packages/components/ng-ovh-doc-url/src/template.html

```diff
@@ -0,0 +1,7 @@
+<a
+    target="_blank"
+    rel="noopener"
+    data-ng-href="{{:: $ctrl.url }}">
+    <span data-ng-transclude></span>
+    <i class="fa fa-external-link" aria-hidden="true"></i>
```

Comment:
```suggestion
    <span class="fa fa-external-link" aria-hidden="true"></span>
    <span class="sr-only">Will open in a new tab</span>
```

Reply:
We can apply it at once when all component will be imported.

Thanks for the suggestions.